src/components/Header/AccountMenu/index.tsx

```diff
@@ -0,0 +1,20 @@
+import AvatarPlaceholder from "@/assets/Images/Avatar.jpg";
+import { Menu } from "lucide-react";
+import Image from "next/image";
+import styles from "./account-menu.module.css";
+
+const AccountMenu = () => {
+  const username = "Kadoodle";
+
+  return ( 
+    <section className={styles.container} >
+      <Menu className={styles.icon} />
+      <span className={styles.greeting}>
+        <span className={styles.username}>{username}</span>
+      </span>
```

Comment:
I think the purpose is to give a difference to the username text but add the "Hey, " 
![image](https://github.com/YARC-Official/yarg.in/assets/59894160/75feed67-0cb0-4058-96b6-c5581a629fe4)
this could solve it:

on the component:
 ```
 <p className={styles.greeting}>
        Hey, <span>{username}</span>
</p>
```

on CSS:

```
.greeting {
  font: 16px;
  font-weight: 400;
  color: #717F9E;
  
> span {
 font-weight: 600;
  color: #FFFFFF;
}


```

Reply:
I see!! I missed that content on CSS.

Reply:
> I think the purpose is to give a difference to the username text but add the "Hey, " ![image](https://private-user-images.githubusercontent.com/59894160/302131809-75feed67-0cb0-4058-96b6-c5581a629fe4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcwODAxNTksIm5iZiI6MTcwNzA3OTg1OSwicGF0aCI6Ii81OTg5NDE2MC8zMDIxMzE4MDktNzVmZWVkNjctMGNiMC00MDU4LTk2YjYtYzU1ODFhNjI5ZmU0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA0VDIwNTA1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRkMjQwNjY2YzU5YjA0ODM4MzlkMDljNmU4M2I5OWUyOGUyMzAzMTliOTRjNDJhNjFlMTdhYjRlYjQ5NDc4NTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.AZsF9iImsvswHnMiBkjKfdh_O2_fYasDX2tPrNmcxu0) this could solve it:
> 
> on the component:
> 
> ```
> <p className={styles.greeting}>
>        Hey, <span>{username}</span>
> </p>
> ```
> 
> on CSS:
> 
> ```
> .greeting {
>   font: 16px;
>   font-weight: 400;
>   color: #717F9E;
>   
> > span {
>  font-weight: 600;
>   color: #FFFFFF;
> }
> ```

that's it, the greeting and username have different colors and font-weight, thus explaining the nested spans but your solution has better html semantic. going to improve it. thanks!